scripts/deblur

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2013, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import click
+
+from skbio.parse.sequences import parse_fasta
+
+from deblur.deblurring import deblur
+
+
+@click.group()
+def deblur_cmds():
+    pass
+
+
+@deblur_cmds.command()
+@click.argument('seqs_fp', required=True,
```

Comment:
@amnona this is requiring always a single filepath; i.e. directories are not supported.